src/NServiceBus.Core/Routing/MessageDrivenSubscriptions/PublisherTableEntry.cs

```diff
@@ -0,0 +1,29 @@
+namespace NServiceBus.Routing.MessageDrivenSubscriptions
+{
+    using System;
+
+    /// <summary>
+    /// Represents an entry in a publisher table.
+    /// </summary>
+    public class PublisherTableEntry
```

Comment:
Probably yes but since it is not on the hot path I don't think making it a struct is worth the effort.